src/styles/blocks/_button.scss

```diff
@@ -0,0 +1,23 @@
+.button{
+  display: flex;
+  width: 166px;
+  height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Use padding instead